packages/cli/README-source.md

```diff
@@ -451,6 +451,9 @@ Also, `authentication.oauth2Config.getAccessToken` has access to the additional
 
 If you define `fields` to collect additional details from the user, please note that `client_id` and `client_secret` are reserved keys and cannot be used as keys for input form fields.
 
+> Note: The Oauth2 `state` param is a [standard security feature](https://auth0.com/docs/secure/attack-protection/state-parameters) that helps ensure that authorization requests are only coming from your servers. Most Oauth clients have support for this and will send back the `state` query param that the user brings to your app. The Zapier Platform performs this check and this required field cannot be disabled. The state parameter is automatically generated by Zapier in the background, and can be accessed at `bundle.inputData.state`. 
```

Comment:
This is a great addition! Small suggestion - I noticed inconsistent capitalization with `OAuth` (i.e. not `Oauth` or `Oauth2`). Ideally, the capitalization would be uniform.

Reply:
Thanks @gregilo! I went through and standardized 😊